Tails specific: we provide our own tails-kexec initscript
(more friendly to ejected CD/USB).

--- chroot.orig/etc/init.d/kexec	2012-09-24 10:05:13.065048881 +0200
+++ chroot/etc/init.d/kexec	2012-09-24 10:03:30.638108333 +0200
@@ -8,1 +8,1 @@
-# Default-Stop:		6
+# Default-Stop:
